packages/clients/textLocator/src/plugins/GeometrySearch/store/actions/setupDrawReaction.ts

```diff
@@ -0,0 +1,35 @@
+import { PolarActionHandler, PolarStore } from '@polar/lib-custom-types'
+import debounce from 'lodash.debounce'
+import { Feature } from 'ol'
+import VectorSource, { VectorSourceEvent } from 'ol/source/Vector'
+import { GeometrySearchGetters, GeometrySearchState } from '../../types'
+
+let debouncedSearchGeometry: PolarActionHandler<
+  GeometrySearchState,
+  GeometrySearchGetters
+>
+
+export function setupDrawReaction(
+  this: PolarStore<GeometrySearchState, GeometrySearchGetters>,
+  { rootGetters, dispatch }
```

Comment:
```suggestion
// Types inferred from PolarActionHandler
export function setupDrawReaction(
  this: PolarStore<GeometrySearchState, GeometrySearchGetters>,
  {
    rootGetters,
    dispatch,
  }: PolarActionContext<GeometrySearchState, GeometrySearchGetters>
```
Together with the import

Reply:
https://github.com/Dataport/polar/pull/115/commits/86b7fe9df8621e50e1682562a65ebb0afed44eb4